### DISCUSSION TITLE
Push pin icon

### DISCUSSION THREAD

**Discussion by Soviut** [tailwindlabs/heroicons#812] [Ideas]
Something like this.

![image](https://user-images.githubusercontent.com/226882/189828473-3bf12c9d-b3b9-496a-bd40-bfcf06eaf400.png)

Or this

![image](https://user-images.githubusercontent.com/226882/189828790-1286924e-9325-4c6f-a672-55658c21513a.png)

**Comment by ndri** [comment]
I added `pin` and `pin-slash` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: 
- https://sidekickicons.com/?icon=pin
- https://sidekickicons.com/?icon=pin-slash
